python/edm4hep/datamodel.py

```diff
@@ -0,0 +1,29 @@
+import ROOT
+res = ROOT.gSystem.Load('libedm4hepDict.so')
+if res < 0:
+    raise RuntimeError('Failed to load libedm4hepDict.so')
+
+kinematics_path = 'edm4hep/utils/kinematics.h'
```

Comment:
kinematics is something used for the tests or provided as default?

Reply:
It's a utility header that we ship by default.

Reply:
thanks